realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -661,6 +1142,16 @@ public static String getTableName() {
                     }
                     reader.endArray();
                 }
+            } else if (name.equals("columnStringList")) {
```

Comment:
print a warning? but it feels will flood user's log. So I guess it is fine as long as we add some javadoc to the `Realm.createObjectFromJson()`

Reply:
done